test/packages/policy_groups/manifest.yml

```diff
@@ -0,0 +1,115 @@
+  kibana.version: '^7.9.0'
+icons:
+  - src: /img/logo_aws.svg
+    title: logo aws
+    size: 32x32
+    type: image/svg+xml
+screenshots:
+  - src: /img/filebeat-aws-cloudtrail.png
+    title: filebeat aws cloudtrail
+    size: 1702x1063
+    type: image/png
+  - src: /img/filebeat-aws-elb-overview.png
+    title: filebeat aws elb overview
+    size: 5120x2704
+    type: image/png
+vars:
+  - name: access_key_id
+    type: text
+    title: Access Key ID
+  - name: secret_access_key
+    type: text
+    title: Secret Access Key
+policy_templates:
+  - name: ec2
+    title: AWS EC2
+    description: Collect logs and metrics from EC2 service
+    inputs:
+      - type: s3
+        title: Collect logs from EC2 service
+        description: Collecting EC2 logs using S3 input
+        policy_group: logs
+        vars:
+          - name: visibility_timeout
+            type: text
+            title: Visibility Timeout
+            description: The duration that the received messages are hidden from subsequent retrieve requests after being retrieved by a ReceiveMessage request.  The maximum is 12 hours.
+          - name: api_timeout
+            type: text
+            title: API Timeout
+            description: The maximum duration of AWS API can take. The maximum is half of the visibility timeout value.
+      - type: aws/metrics
+        title: Collect metrics from EC2 service
+        description: Collecting EC2 metrics using AWS CloudWatch
+        policy_group: metrics
+        vars:
+          - name: endpoint
+            type: text
+            title: Endpoint
+            default: "amazonaws.com"
+            description: URL of the entry point for an AWS web service.
+    icons:
+      - src: /img/aws-ec2.svg
+        title: AWS EC2 logo
+        size: 32x32
+        type: image/svg+xml
+    screenshots:
+      - src: /img/aws-ec2-overview.png
+        title: AWS EC2 Overview
+        size: 1702x1063
+        type: image/png
+    vars:
+      - name: ec2_foobar_property
+        type: bool
+        title: EC2 Specific Property
+  - name: dynamodb
+    title: AWS DynamoDB
+    description: Collect logs and metrics from DynamoDB service
+    categories:
+      - datastore
+    inputs:
+      - type: s3
+        title: Collect logs from the DynamoDB service
+        description: Collecting DynamoDB logs using S3 input
+        policy_group: logs
+      - type: aws/metrics
+        title: Collect metrics from DynamoDB service
+        description: Collecting DynamoDB metrics using AWS CloudWatch
+        policy_group: metrics
+    icons:
+      - src: /img/aws-dynamodb.svg
+        title: AWS DynamoDB logo
+        size: 32x32
+        type: image/svg+xml
+    screenshots:
+      - src: /img/aws-dynamodb-overview.png
+        title: AWS DynamoDB Overview
+        size: 1702x1063
+        type: image/png
+    vars:
+      - name: dynamodb_foobar_property
+        type: bool
+        title: DynamoDB Specific Property
+policy_groups:
```

Comment:
we should call it something else, since effectively we're grouping inputs, the name should reflect that.

Reply:
we're still stuck on aligning names, so apologies in advance if such comments may disrupt this PR. But we're getting there.

Reply:
How about we keep `groups` here?

Reply:
I'm -1 for calling it just `groups` as that's rather generic; I'd prefer to call it something more descriptive and/or specific.  

Maybe `input_groups` per @sorantis [comment](https://github.com/elastic/package-spec/pull/137/files#r579334776)?

Yay, naming 😄.

Reply:
Renamed to `input_groups`.